sensors/Vision360.py

```diff
@@ -1,24 +1,58 @@
 #!/usr/bin/env python3
+import asyncio
+import websockets
+import json
 import math
 from sensors.Sensor import Sensor
 from world.ObjectType import *
 from world.WorldObject import *
 
+import sys
+sys.path.insert(1, "omnicam-zero") # Horrible hacky bit to import the omnicam-zero library because it has a dash in it.
+from protocol import Mode, COMMUNICATION_PORT, REMOTE_ADDR
+
 
 class Vision360(Sensor):
     """class for interfacing with the 360 vision system"""
 
-    def __init__(self):
+    async def connect(self, remote):
+        self.websocket = await websockets.connect(f"ws://{REMOTE_ADDR if remote else 'localhost'}:{COMMUNICATION_PORT}")
+        try:
+            while True:
+                data= await self.websocket.recv()
```

Comment:
Am I reading this correctly that the piwarsimulator program is now listening on a port? Or is this still for going out to make a connection? Or it's formed a connection and is constantly waiting to recv data on that connection whenever the other side is going to send it?

I'm really concerned about the asyncio code interrupting something time critical - a serial transmission to the motor driver board, a decision to not crash into a wall, etc.